projects/driver_display/inc/driver_display_config.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+// Configuration module for the can_slave
+// Defines settings for uart, can_hw and can_uart and loads them
+
+#include "can.h"
+#include "can_hw.h"
+#include "can_uart.h"
+#include "uart.h"
+
+#define DRIVER_DISPLAY_CONFIG_UART_BAUDRATE 115200
```

Comment:
If you're defining all these here, why not just stack-allocate your settings? It seems pointless to statically allocate settings.